integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/resources/BridgeResource.java

```diff
@@ -95,7 +95,7 @@ public static void deleteBridge(String token, String bridgeId) {
         deleteBridgeResponse(token, bridgeId)
                 .then()
                 .log().ifValidationFails()
-                .statusCode(202);
+                .statusCode(400);
```

Comment:
Please revert this change, as it may be misleading. IMHO it is better to wait until the code is properly integrated.

Reply:
Reverted